fed/api.py

```diff
@@ -41,8 +41,9 @@ def init(
     tls_config: Dict = None,
     logging_level: str = 'info',
     cross_silo_grpc_retry_policy: Dict = None,
-    cross_silo_send_max_retries: int = None,
     cross_silo_serializing_allowed_list: Dict = None,
+    cross_silo_send_options: Dict = None,
```

Comment:
`cross_silo_send_options` equals actor options?

Reply:
yes, the "cross_silo" prefix may be more clear to user, what do u think ?

Reply:
Naming is ok, but `cross_silo_send_options` == actor.options is worth discussing.